README.md

```diff
@@ -52,13 +50,13 @@ As a Maker
 So that I can see when people are doing things
 I want to see the date the message was posted
 ```
-(Hint the database table will need to change to store the date too)
 
 ```
 As a Maker
 So that I can easily see the latest peeps
 I want to see a list of peeps in reverse chronological order
 ```
+This user story was not yet built
```

Comment:
What would you plan be, if you were going to add this feature?  How would it work?